### PR TITLE
Update high-resolution-photos.md

### PR DESCRIPTION
**Pull request by sowjanya365**
Issue #2716 - Missing exchange versions are updated.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbServer/deploy/integrate-with-exchange-server/high-resolution-photos.md [hunk 1]
```diff
@@ -11,14 +11,14 @@ ms.prod: skype-for-business-itpro
 localization_priority: Normal
 ms.collection: IT_Skype16
 ms.assetid: 995da78a-dc44-45a3-908d-16fe36cfa0d9
-description: "Summary: Configure the use of high-resolution photos in Exchange Server 2016 or Exchange Server 2013 and Skype for Business Server."
+description: "Summary: Configure the use of high-resolution photos in Exchange Server 2019 or Exchange Server 2016 or Exchange Server 2013 or Exchange Online and Skype for Business Server."
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/deploy/integrate-with-exchange-server/high-resolution-photos.md]:
```suggestion
description: "Summary: Configure the use of high-resolution photos in Exchange Server 2019, Exchange Server 2016, Exchange Server 2013, or Exchange Online and Skype for Business Server."
```

File: Skype/SfbServer/deploy/integrate-with-exchange-server/high-resolution-photos.md [hunk 2]
```diff
@@ -11,14 +11,14 @@ ms.prod: skype-for-business-itpro
...
 ---
 
 # Configure the use of high-resolution photos in Skype for Business Server
  
-**Summary:** Configure the use of high-resolution photos in Exchange Server 2016 or Exchange Server 2013 and Skype for Business Server.
+**Summary:** Configure the use of high-resolution photos in Exchange Server 2019 or Exchange Server 2016 or Exchange Server 2013 or Exchange Online and Skype for Business Server.
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/deploy/integrate-with-exchange-server/high-resolution-photos.md]:
```suggestion
**Summary:** Configure the use of high-resolution photos in Exchange Server 2019, Exchange Server 2016, Exchange Server 2013, or Exchange Online and Skype for Business Server.
```

File: Skype/SfbServer/deploy/integrate-with-exchange-server/high-resolution-photos.md [hunk 3]
```diff
@@ -11,14 +11,14 @@ ms.prod: skype-for-business-itpro
...
   
-In Skype for Business Server photos can be stored in a user's Exchange Server 2016 or Exchange Server 2013 mailbox, which allows for photo sizes up to 648 pixels by 648 pixels. In addition, Exchange Server can automatically resize these photos for use in different products as needed. Typically that means three different photo sizes and resolutions:
+In Skype for Business Server photos can be stored in a user's Exchange Server 2019 or Exchange Server 2016 or Exchange Server 2013 or Exchange Online mailbox, which allows for photo sizes up to 648 pixels by 648 pixels. In addition, Exchange Server can automatically resize these photos for use in different products as needed. Typically that means three different photo sizes and resolutions:
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/deploy/integrate-with-exchange-server/high-resolution-photos.md]:
```suggestion
In Skype for Business Server, photos can be stored in a user's Exchange Server 2019, Exchange Server 2016, Exchange Server 2013, or Exchange Online mailbox, which allows for photo sizes up to 648 pixels by 648 pixels. In addition, Exchange Server can automatically resize these photos for use in different products as needed. Typically that means three different photo sizes and resolutions:
```

**Comment by scanum** [comment]
@LanaChin Following up on this PR, please let me know if there is anything I can do to assist. Thanks!

**Comment by LanaChin** [comment]
@sowjanya365, please commit the copy edit changes made by  
@JohanFreelancer9 JohanFreelancer9.  I'll merge after those changes are commited. Thank you.

**Comment by sowjanya365** [comment]
> @sowjanya365, please commit the copy edit changes made by
> @JohanFreelancer9. I'll merge after those changes are commited. Thank you.

Hi @LanaChin 

Done. Thank you! :)

**Pull request merged**